contracts/LiquidDemocracy.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity ^0.4.4;
+
+contract LiquidDemocracy {
+
+    // This is the type for a single option shown in a ballot
```

Comment:
Do you mind specify here that for now, only Yes/No are allowed? that also exemplifies what a proposal could be